kw/platform/requests/__init__.py

```diff
@@ -0,0 +1,9 @@
+try:
+    from requests import *
+except ModuleNotFoundError:
+    raise ImportError("missing module 'requests'")
+
+from ..requests_utils import patch_with_user_agent, set_user_agent
+
+
+patch_with_user_agent()
```

Comment:
Oh waiiiit, I just noticed this is wrong 😧 

This patches the global requests library, which we wanted to keep intact. We should be patching specifically `kw.platform.requests`.

Reply:
Oh I misunderstood that completely I thought that you'll have an option to either import `kw.platform.requests` that will patch the library or original `requests` that  you'll be able to patch.